core/jobs_test.py

```diff
@@ -657,6 +731,89 @@ def test_validity_of_each_continuous_computation_class(self):
                 for superclass in allowed_base_batch_job_classes]))
 
 
+class BaseMapReduceJobManagerForContinuousComputationsTests(
+        test_utils.GenericTestBase):
+
+    def test_raise_error_with_get_continuous_computation_class(self):
+        with self.assertRaisesRegexp(
+            NotImplementedError,
+            re.escape(
+                'Subclasses of '
+                'BaseMapReduceJobManagerForContinuousComputations must '
+                'implement the _get_continuous_computation_class() method.')):
+            (
```

Comment:
I don't think you need the parens, here and below. You could just deindent the next line by 4?

Reply:
This doesn't work. If I need to break the line after ".", I need to add parens